.npmrc

```diff
@@ -0,0 +1 @@
+@openolitor-core:registry=https://registry.npmjs.org/
```

Comment:
Is this changing the location of the openolitor-core?? we used to fetch it from github, right ?

Reply:
No, not at all. It tells where to fetch the dependencies from for this project. For most people this does not matter. Only for us or others that have a private registry (nexus). Without this, we get all nexus links in the package-lock.json file.